helm_inspect/utils/constant.py

```
@@ -0,0 +1,66 @@
+"""
+Slack API URL for completing the file upload.
+"""
+
+SLACK_MESSAGE_URL = "https://slack.com/api/chat.postMessage"
+"""
+Slack API URL for posting messages.
+"""
+
+BASE_DIR: Path = (
+    Path.home() / ".helminspect"
+    if not "HI_BASE_DIR" in os.environ
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)